transient-aws/hive-example/cluster_bare_amis.conf

```diff
@@ -0,0 +1,194 @@
+    -----END RSA PRIVATE KEY-----
+"""
+}
+
+#
+# A list of instance types to use for group of nodes or management services
+#
+
+instances {
+    master {
+        type: c4.xlarge
+        image: ami-775e4f16 [REPLACE_ME]
+        instanceNamePrefix: master-cdh
+        tags {
+            owner: ${?USER}
+        }
+    }
+    worker {
+        type: r3.xlarge
+        image: ami-775e4f16 [REPLACE_ME]
+        instanceNamePrefix: worker-cdh
+        tags {
+            owner: ${?USER}
+        }    
+    }
+
+    worker-spot{
+        type: r3.xlarge
+        image: ami-d2c924b2 [REPLACE_ME]
+        instanceNamePrefix: worker-spot
+        tags {
+            owner: ${?USER}
+        }
+    }
+}
+
+
+#
+# Configuration for Cloudera Manager. Cloudera Director can use an existing instance
+# or bootstrap everything from scratch for a new cluster
+#
+
+cloudera-manager {
+
+    instance: ${instances.master} {
+        tags {
+            application: "Cloudera Manager 5"
+        }
+    }
+
+    #
+    # Automatically activate 60-Day Cloudera Enterprise Trial
+    #
+
+    enableEnterpriseTrial: true
+
+    repository: "http://archive.cloudera.com/cm5/redhat/7/x86_64/cm/5.8/"
+    repositoryKeyUrl: "http://archive.cloudera.com/cm5/redhat/7/x86_64/cm/RPM-GPG-KEY-cloudera"
+
+}
+
+#
+# Cluster description
+#
+
+cluster {
+
+    products {
+      CDH: 5.8 # includes Hive and Spark
+    }
+
+
+    # S3 Configurations
+    configs {
+      HDFS {
+        core_site_safety_valve: """
+            <property>
+                <name>fs.s3a.access.key</name>
+                <value>REPLACE_ME</value>
+            </property>
+            <property>
+                <name>fs.s3a.secret.key</name>
+                <value>REPLACE_ME</value>
+            </property>
+            <property>
+                <name>fs.s3a.block.size</name>
+                <value>134217728</value>
+            </property>
+            <property>
+              <name>fs.s3a.server-side-encryption-algorithm</name>
+              <value>AES256</value>
+            </property>
+            <property>
+              <name>fs.s3a.connection.ssl.enabled</name>
+              <value>true</value>
+              <description>Enables or disables SSL connections to S3.</description>
+            </property>
+        """
+      }
+    }
+
+    parcelRepositories: ["http://archive.cloudera.com/cdh5/parcels/5.8/"]
+
+    services: [HDFS, YARN, HIVE, SPARK_ON_YARN, OOZIE]
+
+    masters {
+      count: 1
+      instance: ${instances.master}
+
+      roles {
+        HDFS: [NAMENODE, SECONDARYNAMENODE]
+        YARN: [RESOURCEMANAGER, JOBHISTORY]
+        HIVE: [HIVESERVER2, HIVEMETASTORE]
+        OOZIE: [OOZIE_SERVER]
+        SPARK_ON_YARN: [SPARK_YARN_HISTORY_SERVER]
+      }
+    }
+
+    workers {
+      count: 2 [REPLACE_ME]
+
+      instance: ${instances.worker}
+
+      roles {
+        HDFS: [DATANODE]
+        YARN: [NODEMANAGER]
+      }
+    }
+
+#    workers-spot {
```

Comment:
Any reason this section is here, given that it is commented out? Can it be removed?